validate.js

```diff
@@ -11,7 +11,7 @@ const NodeRSA = require('node-rsa');
  * @return {function}            Returns validator function
  */
 module.exports = function (s3oPublicKey) {
-	return function (key, token) {
+	return function (key, signedToken) {
```

Comment:
is there also a less generic name than 'key' that can be used here

Reply:
I was trying to think of one...